### DISCUSSION TITLE
Data Storage folder / drive

### DISCUSSION THREAD

**Discussion by HuberMax2** [kanriapp/kanri#529] [Q&A]
Thanks for the great app.
I have the following questions:
Where is the data stored? 
Is it possible to set the storage location to a cloud drive (Nextcloud)?

Thanks for the help in advance
Max

**Comment by trobonox** [comment]
Hi, unfortunately at the moment the storage location is fixed and cannot be changed, this is a restriction of `tauri-plugin-store` which I am using under the hood.
I have plans to implement some custom logic to allow changing paths, but this will only happen in the future versions (probably only towards version 0.8/0.9, so it will take some time).

**Reply by HuberMax2** [reply]
Thank you

**Comment by lennybacon** [comment]
Sync issues aside... There is the option to use a hardlink or junction to redirect the store location to another folder.

Where is the data stored?

**Reply by trobonox** [reply]
You can find the storage location in the FAQ section of the docs: https://www.kanriapp.com/getting-started/faq/#where-is-my-data-saved-can-i-change-the-location

**Reply by lennybacon** [reply]
Thanks!
